ParalayoutTests/GeometryAdditionsTests.swift

```diff
@@ -127,5 +127,18 @@ class GeometryAdditionsTests: XCTestCase {
         XCTAssert(sample.slice(from: .maxYEdge, amount: 100) == (sample,
                                                                  CGRect(x:  0,   y:  0, width: 100, height:   0)))
     }
+
+    // MARK: - Private Methods
+
+    private func screensToTest() -> [UIScreen] {
```

Comment:
Small nit: 

Since we have this same function in HairlineTests.swift, maybe we could put this function in a separate utilities file, i.e. UIScreen+Utilities.swift 
```
func screensToTest() -> [UIScreen]
```

Reply:
We're going to remove `HairlineTests` shortly as part of #24, so that duplication will be short-lived.